### ISSUE TITLE
  Corresponding problem of kitti and vkitti

### ISSUE THREAD

**Issue by name333** [lyndonzheng/Synthetic2Realistic#24]
Hello, I have some questions I would like to ask you. In your paper, the results are very good. In the course of my training, vkitti and kitti are randomly selected, and the evaluation effect is not as high as yours. So I would like to ask, when you are training, is there a certain rule correspondence between kitti and vkitti, for example, in the training process, the kitti and vkitti are similarly extracted.
Or a completely unordered random extraction of both pictures.

**Comment by lyndonzheng** [comment]
Your evaluation results are close to the results, or have a large gap to the results reported in our paper? Because each time I train the model, the evaluation results also have a little variation, sometimes are worse, sometimes are better. We think that's variation comes from the translation network that generates different images during the training. As for the KITTI and vKITTI, we use the random order without paired  information.

**Comment by name333** [comment]
The results of my assessment:
Abs Rel、   Sq Rel、  RMSE  、RMSE(log) 、∂<1.25 、 ∂<1.25^2 、 ∂<1.25^3
0.1966 、 1.6097 、 5.4739 、 0.2860       、  0.7128   、0.8794 、     0.9493
You can see that the results are close to your paper, but there are still some gaps. Can you ask what other skills are available during the training phase, which can be closer to your results?

**Comment by lyndonzheng** [comment]
I think this accuracy has a little gap between our results. For the pretrained model that I provided in google drive, I just train it end-to-end for once, and get a very close results to the original paper. Could you provide the training loss? Besides, how many epoch are you used?

**Comment by name333** [comment]
yes,My loss is probably: img_rec: 3.0, img_G: 0.3, img_D: 0.2, lab_s: 6.0; F_G0.04; f_d0.2, lab_smooth: 0.02, fluctuating above and below. epoch=10, iters22000.

**Comment by lyndonzheng** [comment]
If you just use the normal images of vkitti, you can try to train with 20 epoch. You can directly --continue_train the current model with another 10 epoch. I think the range of your loss is correct.

**Comment by name333** [comment]
Thank you for your reply, let me try it.
